src/components/Carousel/types.ts

```diff
@@ -40,9 +40,9 @@ export interface CarouselProps {
     onClickItem: (index: number, item: React.ReactNode) => void;
     onClickThumb: (index: number, item: React.ReactNode) => void;
     onChange: (index: number, item: React.ReactNode) => void;
-    onSwipeStart: (event: React.TouchEvent) => void;
-    onSwipeEnd: (event: React.TouchEvent) => void;
-    onSwipeMove: (event: React.TouchEvent) => boolean;
+    onSwipeStart: (event: React.TouchEvent | React.MouseEvent) => void;
```

Comment:
Not sure if this is correct typings, but it aligns with how it's actually typed in the library it's later used in. 
Opted for not using the `SwipeEvent` to not add dependency to external types in the exported type.